projects/github.com/toy/blueutil/package.yml

```diff
@@ -0,0 +1,20 @@
+distributable:
+  url: git+https://github.com/toy/blueutil
+  ref: v{{version}}
+  strip-components: 1
+versions:
+  github: toy/blueutil
+platforms:
+  - darwin/aarch64
+  # Set to build with SDK=macosx10.6
+  # - darwin/x86-64
```

Comment:
Unfortunately, the test fails due to the SDK version on the Workflow virtual machine. I have x86-64 SDK 12.3 everything is ok